scripts/bluetooth/mesh/mesh_dfu_metadata.py

```diff
@@ -348,16 +346,18 @@ def parse_comp_data(elf_path, kconfigs):
     Parameters:
         kconfigs (KConfig): A KCoonfig object representing Kconfig options used for the firmware to compile with
     Returns:
-        Parsed Composition data
+        List of parsed composition data
     """
     try:
-        addr = find_comp_data_from_dwarf(elf_path)
-        return read_comp_data(elf_path, addr, kconfigs)
-    except Exception as err :
+        addrs = find_comp_data_from_dwarf(elf_path)
+        return [comp
+                for addr in addrs
+                for comp in read_comp_data(elf_path, addr, kconfigs)]
+    except Exception as err:
         raise Exception("Failed to extract composition data from .elf file") from err
 
 def encoded_metadata_get(version, comp, binary_size):
-    core_type = 1 # FIX ME: For now, hardcoded to application core
+    core_type = 1 # FIXME: For now, hardcoded to application core
```

Comment:
If you really want to fix it, then it should have JIRA ticket. Otherwise, there is no sense in such comments. It will be permanent FIXME.

Reply:
This was here before this PR, I simply removed the space. @Andrewpini : since you added this, you can decide whether to add a JIRA issue or if I should remove it here.

Reply:
I think we can just remove the FIXME. We will most likely have to address this at some point, but the comment is not neccesary.

Reply:
Shouldn't we include core_type into json file even though it is hardcoded now?

Reply:
You mean as a separate field? Or to keep it in the encoded metadata? If you mean the latter, I'm only planning to remove the comment here, keep the rest. If you mean to add a separate field to the json object, please add a separate issue for this. It's not part of making the script handle multiple comp data which is what I'm fixing here.

Reply:
NCSDK-26810